site/docs/v1/tech/apis/_application-response-body-base.adoc

```diff
@@ -386,6 +386,12 @@ Indicates if this application enables WebAuthn workflows based on the configurat
 [field]#{base_field_name}.webAuthnConfiguration.reauthenticationWorkflow.enabled# [type]#[Boolean]# [since]#Available since 1.41.0#::
 Whether the WebAuthn reauthentication workflow is enabled for this application. This overrides the tenant configuration. Has no effect if [field]#application.webAuthnConfiguration.enabled# is `false`.
 
+ifdef::include_total[]
+[field]#total# [type]#[Integer]# [since]#Available since 1.44.0#::
```

Comment:
This response field is on a new API rather than being an update to an existing API. I don't think this needs the `since` message.